Sources/SwiftInspectorVisitors/Tests/TypeSyntaxVisitorSpec.swift

```diff
@@ -112,11 +112,6 @@ final class TypeSyntaxVisitorSpec: QuickSpec {
         }
         """
 
-        it("detects the type name") {
```

Comment:
The `TypeSyntaxAnalyzer` is initializer with a type name. As a result, we no longer return the type name in the analyzed result (i.e. we removed `TypeProperties`). And so we no longer need these tests.